gradle.properties

```diff
@@ -14,7 +14,7 @@ maven_group = pl.telvarost
 archives_base_name = mojangfixstationapi
 
 # Dependencies
-stapi_version=2.0-alpha.2
+stapi_version=2.0-alpha.2.3
```

Comment:
Isn't StationAPI 2.3 broken in prod right now? Shouldn't this be 2.1?

Reply:
It doesn't make a difference, as the things changed in 2.3 versus 2.1 are not touched by this mod, so pretty much the only difference is the runtime version of stationapi as all API things remain the same.

Reply:
and in dev it works fine

Reply:
I usually like to set it to whatever, players will actually experience. So I will probably revert it back to 2.1 until StationAPI stabilizes

Reply:
Sure, it doesn't make much of a difference to me